android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestSuggestionProvider.kt

```diff
@@ -18,16 +18,17 @@ import java.util.UUID
  *
  * @param resources Your application's [Resources] instance.
  * @param loadUrlUseCase A use case that loads a suggestion's URL when clicked.
- * @param includeSponsoredSuggestions Whether to return suggestions for sponsored content.
- * @param includeNonSponsoredSuggestions Whether to return suggestions for web content.
+ * @param providers The [SuggestionProvider]s to query for search suggestions.
  * @param suggestionsHeader An optional header title for grouping the returned suggestions.
  * @param contextId The contextual services user identifier, used for telemetry.
  */
 class FxSuggestSuggestionProvider(
     private val resources: Resources,
     private val loadUrlUseCase: SessionUseCases.LoadUrlUseCase,
-    private val includeSponsoredSuggestions: Boolean,
-    private val includeNonSponsoredSuggestions: Boolean,
+    private val providers: List<SuggestionProvider> = listOf(
+        SuggestionProvider.AMP,
+        SuggestionProvider.WIKIPEDIA,
+    ),
```

Comment:
Do we intentionally want to include a sponsored provider as a default here?

Reply:
Oooh, no, good point! Apart from a few tests, we're always passing a list of `providers`, so we can drop the default entirely.